src/Std/Data/DTreeMap/RawLemmas.lean

```diff
@@ -0,0 +1,56 @@
+/-
+Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel, Paul Reichert
+-/
+prelude
+import Std.Data.DTreeMap.Internal.Lemmas
+import Std.Data.DTreeMap.Raw
+
+/-!
+# Dependent tree map lemmas
+
+This file contains lemmas about `Std.Data.DTreeMap.Raw`. Most of the lemmas require
+`TransCmp cmp` for the comparison function `cmp`.
+-/
+
+set_option linter.missingDocs true
+set_option autoImplicit false
+
+open Std.DTreeMap.Internal
+
+universe u v
+
+namespace Std.DTreeMap.Raw
+
+variable {α : Type u} {β : α → Type v} {cmp : α → α → Ordering} {t : DTreeMap.Raw α β cmp}
+
+@[simp]
+theorem isEmpty_empty : (empty : DTreeMap.Raw α β cmp).isEmpty :=
+  Impl.isEmpty_empty
+
+@[simp]
+theorem isEmpty_emptyc : (∅ : DTreeMap.Raw α β cmp).isEmpty :=
+  Impl.isEmpty_empty
```

Comment:
I believe we discussed that we wanted to rewrite `empty`  to `∅`, so the lemmas in terms of `empty` should no be necessary?